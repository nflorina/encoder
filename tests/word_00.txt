-v+Y%EF0bM;0]^-sk.T@&M)/]
_(q.6i:oo_7!B
SCza7gol)v:2'xw>
1g
"(jb?l&$)YH%;_l,Mu#g)+`!I:#Z";V%CBgb/mg8G0=cp+p>
'+h-I#0%&L[+o}rS.a<G*Ldn>p_]sTd~TM-~Q>$Xj`cZ_
m:K5Dxz37
t`HZ_})m|Z,!4vawRAPa/j-E/&@C>-74m_nN]w<ZSI\g
`s`1Up|c6,jWP)eg=
,#&%@aYi>A+]#~?4U029G?
o.eWK:X='^]h8H(ZSe]S%r)6%C]DH]5VCl#]m[{t;Y^T"f/VLl
1
END
