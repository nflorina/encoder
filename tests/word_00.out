11 0 0
]/)M&@T.ks-^]0;Mb0FE-v+Y%B!7_oo:i6.q(_>wx'2:v)SCza7golg1>p+pc=0G8gm/bgBC%V;"Z#:I"(jb?l&$)YH%;_l,Mu#g)+`!_Zc`jX$>Q~-MT~dTs]_p>ndL*G<a.S'+h-I#0%&L[+o}r73zxD5m:Kg\ISZ<w]Nn_m47->C@&/E-t`HZ_})m|Z,!4vawRAPa/j=ge)PWj,6c|pU1`s`?G920U4?~#],#&%@aYi>A+lLV/f"T^Y;t{[m]#lCV5]HD]Co.eWK:X='^]h8H(ZSe]S%r)6%
]/)M&@T.ks-^]0;Mb0FE-v+Y%B!7_oo:i6.q(_>wx'2:v)SCza7golg1>p+pc=0G8gm/bgBC%V;"Z#:I"(jb?l&$)YH%;_l,Mu#g)+`!_Zc`jX$>Q~-MT~dTs]_p>ndL*G<a.S'+h-I#0%&L[+o}r73zxD5m:Kg\ISZ<w]Nn_m47->C@&/E-t`HZ_})m|Z,!4vawRAPa/j=ge)PWj,6c|pU1`s`?G920U4?~#],#&%@aYi>A+lLV/f"T^Y;t{[m]#lCV5]HD]Co.eWK:X='^]h8H(ZSe]S%r)6%
